OrbitCore/CrashHandler.cpp

```diff
@@ -14,6 +16,10 @@
 #include "TcpClient.h"
 #include "Version.h"
 
+// TODO: Remove this flag once we have a dialog with user
+ABSL_FLAG(bool, upload_dumps_to_server, false,
```

Comment:
Nit: To make CrashHandler testable without messing with ABSL_FLAG it's better to put the Flag in `OrbitQt/main.cpp` and pass the value via CrashHandler's constructor to `SetUploadsEnabled`